Orange/widgets/cpewidget.py

```diff
@@ -32,7 +36,11 @@ def sync_properties(self):
         # to the widget
         for n, p in self.node.ports(direction='IN*', editable=True).items():
             if n not in settings or settings[n] is None:
-                super().__setattr__(n, getattr(self.node, n))
+                value = getattr(self.node, n)
+                if (isinstance(p, ListPort) or isinstance(p, IntPort)) and value is None:
```

Comment:
Ok, there are actually 2 JIRA items, one that talks about ListPort's interpretation of None as [None], and another one that talks about Line-edit ports (that is, all ports that render in form of a line edit) and give IntPort as an example. The full list is: IntPort, FloatPort, StringPort, ListPort.  

Now it would be better if we didn't have to check the port type since it makes it more messy, and BoolPort and EnumPort shouldn't produce None values or allow entry of "(default)" anyway, so it seems justified to not check the port type at all. If it ever occurred at some point in the future that somehow a None value travels into an EnumPort/BoolPort, that could be fixed elsewhere (since it wouldn't render properly anyway, with or without value conversion logic, so needs to be fixed someplace else).